main.cpp

```diff
@@ -475,7 +484,15 @@ class auto_charger { // Variables Half-Implemented (Not Thermistors ADC)
             send_heartbeat();
     }
     void send_heartbeat() {                                                    /* Creates the message to send to the robot using the "compose" function below */
-        uint8_t buf[8], param[3]{++heartbeat_counter, static_cast<uint8_t>(sw.read()), rsoc}; // Message composed of 8 bytes, 3 bytes parameters -- Declaration
+        uint8_t sw_state{0};
+
+        if(is_connected()){
+            sw_state = 1;
+        }else{
+            sw_state = 0;
+        }
```

Comment:
スペースの入れ方を他と揃える。
Align the way you put spaces with others.

```suggestion
        if (is_connected()) {
            sw_state = 1;
        } else {
            sw_state = 0;
        }
```

Reply:
修正してコミットしました。

Fixed and committed.